cmd/commands/service_test.go

```diff
@@ -663,3 +664,73 @@ func pathAwareHttpServer(path string, pathMatchedChannel chan<- bool) net.Listen
 	}))
 	return listener
 }
+
+var _ = Describe("PrintCurlHttpErrors", func() {
+	var (
+		input  string
+		output string
+	)
+
+	JustBeforeEach(func() {
```

Comment:
purely personal opinion: I tend to avoid `JustBeforeEach` as it makes the reading less linear but the setup is simple enough at this point.